superset/assets/src/dashboard/components/SliceAdder.jsx

```diff
@@ -162,17 +168,27 @@ class SliceAdder extends React.Component {
         useEmptyDragPreview
       >
         {({ dragSourceRef }) => (
-          <AddSliceCard
-            innerRef={dragSourceRef}
-            style={style}
-            sliceName={cellData.slice_name}
-            lastModified={
-              cellData.modified ? cellData.modified.replace(/<[^>]*>/g, '') : ''
-            }
-            visType={cellData.viz_type}
-            datasourceLink={cellData.datasource_link}
-            isSelected={isSelected}
-          />
+          <CellMeasurer
+            cache={cache}
+            columnIndex={0}
+            key={key}
+            parent={parent}
+            rowIndex={index}
+          >
+            <AddSliceCard
+              innerRef={dragSourceRef}
+              style={style}
+              sliceName={cellData.slice_name}
+              lastModified={
+                cellData.modified
+                  ? cellData.modified.replace(/<[^>]*>/g, '')
```

Comment:
is this expensive? should we do this once or is it fine to do each render?

Reply:
fixed, parse text when populate sliceEntity.